server/app.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import logger from 'morgan';
+
+const app = express();
+const http = require('http');
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({
+  extended: true,
+}));
+
+
+app.all('*', (req, res) => {
+  res.status(404).send('The page you are looking for was not found');
+});
+
+const port = parseInt(process.env.PORT, 10) || 8000;
+app.set('port', port);
+
+const server = http.createServer(app);
```

Comment:
@Raymond-Osy Please, can you explain why this is needed?

Reply:
On line 19, the port is set to an environment variable, if none is found, it uses the specified port, in my case port 8000. On line 22, i simply created the server using the express application.

Reply:
@Raymond-Osy To use express and http.createServer in the same application, as they fit the same purpose but express is a more useful abstraction over http.
In express, listen boots the app. Before booting, you need to setup handlers and middlewares for routes (i.e. get, post, use) which you have already done. So in this case using listen is enough.

Reply:
Thanks for the feedback. I have acted on it. Can i go ahead and merge?